web-development/age-calculator/documents/age-calculator.css

```diff
@@ -0,0 +1,22 @@
+body {
+    background: beige;
+    font-family: "Raleway";
+}
+
+.container input {
+    height: 30px;
+    margin: 0 5px;
+    border: 1px solid gray;
+    padding: 0 10px;
+    outline: 0px;
+    width: 100px;
+}
+
+.container input:hover, .container input:focus {
```

Comment:
write them in multi-lines it's more readable:
```
.container input:hover,
.container input:focus {
...
```

Reply:
sure